client/etcd/options.go

```diff
@@ -35,14 +35,14 @@ type Options interface {
 	Zone() string
 	SetZone(z string) Options
 
-	ServiceInitTimeout() time.Duration
-	SetServiceInitTimeout(timeout time.Duration) Options
+	Service() string
```

Comment:
Yea, just feels weird having a type for precisely these fields and not using it. Why is it valid for these options to not have an env/zone - does that query multiple zones/envs for properties?